src/model/differ.js

```diff
@@ -243,6 +246,21 @@ export default class Differ {
 		return result;
 	}
 
+	/**
+	 * Checks whether some of buffered marker can affect the data model.
+	 *
+	 * @returns {Boolean} `true` if buffered markers can change the data model.
+	 */
+	containsMarkerAffectingData() {
```

Comment:
Do we need it? I feel that we'll use it in one place. Instead, I feel that it'd be good to have one differ's method which will do all this: `isBatchAffectingData( writer.batch ) || this.differ.containsMarkerAffectingData()`.

Reply:
What do you mean exactly? Because the `Differ` knows nothing about batches. I can check every operation that passes to the `bufferOperation()` and from that point, I can mark that some operation will change the data model. But I'm not sure if you mean this.

Reply:
I don't know relations between Differ and batches, so I may not make much sense. But what I noticed is that you created a method which will only be used once in its lifetime. It's not very useful, it checks just one thing and is used to do a far more useful thing (check if the data will change), but itself... it's just useless.

So, we have two options:

1. Make it protected. 
2. Expose some useful functionality in the `Differ` – e.g. a method which could check if the data will change. I'm not sure if this is doable, but I thought that perhaps `isBatchAffectData()` could be moved here. Or maybe there's some other simple solution.

Reply:
I think that I can check differs' changed elements and if there is a changed element or marker that affects data, I can say that buffered changes affect data (I'll ask @scofalik tomorrow to be sure it's correct). But how to call that method? `doChangesAffectData`, `areChangesAffectingData`. I'm not sure about the batch in the method name. It looks like an expected parameter and the `Differ` doesn't use the `batch` or the `delta` terminology.

Reply:
`willChangeData()`?

Reply:
Or `willDataChange()`... not sure which sounds better.

Reply:
`willDataChange` sound good to me. Thanks!

Reply:
AFAIC understand you want to have a `Differ` method that will tell you if anything in data changed? @ma2ciek you can do it as you said - check `_changesInElement.size`. For markers data you will need to iterate through `_changedMarkers` and see if there are any markers that change model. Pretty straightforward.